src/lib/prepareOas.ts

```diff
@@ -211,6 +211,34 @@ export default async function prepareOas(
     debug('spec bundled');
   }
 
+  // Checks to see if the selected file is a postman collection and throws a warning in the console depending on the command.
+  if (definitionVersion.specification === 'postman') {
+    switch (command) {
+      case 'openapi':
```

Comment:
great job here with this `if`/`switch` logic — super readable and it's in the perfect spot 👏🏽